core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.component.html

```diff
@@ -1,190 +1,211 @@
+          </span>
         </div>
-        <div class="topic-nav-dropdown-icon" ng-click="toggleNavigationOptions()">
+        <div class="topic-nav-dropdown-icon"
+             (click)="toggleNavigationOptions()">
           <i class="fa fa-caret-down"></i>
         </div>
       </div>
     </div>
     <div class="topic-nav-dropdown-container save-options-dropdown">
-      <div class="topic-nav-topic-nav-dropdown-options" ng-if="showTopicEditOptions">
-        <div class="topic-nav-dropdown-option" ng-click="discardChanges()">
+      <div class="topic-nav-topic-nav-dropdown-options" *ngIf="showTopicEditOptions">
+        <div class="topic-nav-dropdown-option" (click)="discardChanges()">
           <i class="material-icons">&#xE254;</i>
           <span>Discard Changes</span>
         </div>
         <div class="topic-nav-dropdown-option">
-          <div ng-if="!topicRights.isPublished()"
-               ng-click="publishTopic()"
-               ng-disabled="isTopicSaveable() || getWarningsCount() > 0 || prepublishValidationIssues.length > 0">
+          <div *ngIf="!topicRights.isPublished()"
+               (click)="publishTopic()"
+               [attr.disabled]="isTopicSaveable() || getWarningsCount() > 0 || prepublishValidationIssues.length > 0">
             <i class="material-icons">&#xE254;</i>
             <span>Publish Topic</span>
           </div>
-          <div ng-click="unpublishTopic()"
-               ng-if="topicRights.isPublished() && topicRights.canPublishTopic()"
-               ng-disabled="isTopicSaveable()">
+          <div (click)="unpublishTopic()"
+               *ngIf="topicRights.isPublished() && topicRights.canPublishTopic()"
+               [attr.disabled]="isTopicSaveable()">
             <i class="material-icons">&#xE254;</i>
             <span>Unpublish Topic</span>
           </div>
         </div>
       </div>
       <div class="topic-nav-dropdown-content right-container">
-        <div class="topic-nav-dropdown-text" ng-click="saveChanges()">
-          <span ng-if="isSaveInProgress()">
+        <div class="topic-nav-dropdown-text"
+             (click)="saveChanges()">
+          <span *ngIf="isSaveInProgress()">
             <span>Saving</span>
           </span>
-          <span ng-if="!isSaveInProgress()">
+          <span *ngIf="!isSaveInProgress()">
             Save Changes
           </span>
-          <span ng-if="getChangeListLength() > 0">
-            (<[getChangeListLength()]>)
+          <span *ngIf="getChangeListLength() > 0">
+            ({{ getChangeListLength() }})
           </span>
         </div>
-        <div class="topic-nav-dropdown-icon" ng-click="toggleTopicEditOptions()">
+        <div class="topic-nav-dropdown-icon"
+             (click)="toggleTopicEditOptions()">
           <i class="fa fa-caret-down"></i>
         </div>
       </div>
     </div>
   </div>
 </div>
 <style>
-  topic-editor-navbar .icon {
+  .icon {
```

Comment:
Should we prepend all class names with `oppia-` to make sure class names are unique?